src/content-handlers/iiif/modules/uv-searchfooterpanel-module/FooterPanel.ts

```diff
@@ -380,6 +381,10 @@ export class FooterPanel extends BaseFooterPanel<
       return true;
     }
 
+    // if only one canvas then allow clicking next result
+    if (totalCanvases === 1) {
```

Comment:
A couple of comments on this:

1.) I'm wondering if this special case is masking a different/larger bug. I'm not sure why we need to account for having just one canvas differently. If there's only one canvas, then shouldn't currentCanvasIndex equal lastSearchResultCanvasIndex? And if that's the case, how is it any different from being on the last page of a multi-page result? If that's not the case, then maybe there's a bug related to determining the current index, or in one of the other comparisons, and we'd be better off fixing that underlying problem instead of adding a workaround at the end.

2.) If we do indeed need to keep this code as-is, some minor stylistic changes are needed -- `return true` is missing a semi-colon, and the closing brace needs to be dropped to the next line. I'm guessing that perhaps a couple of characters were accidentally deleted here, but Javascript is tolerant enough that it works anyway. :-)

Reply:
Thanks for taking a look Demian. This did cross my mind, but currently the only logic that activates the next button in the first instance is the final return statement of this method (now line 388). The block above it is relevant only when an annotation is 'current' (after 'next' is pressed for the first time), and then it's checking that it is neither the last page or the last annotation. So my change is essentially just adding another condition to check for along with line 388. This is hard to explain! But we can discuss on the next UV call.

Reply:
The more streamlined solution worked so I've edited the PR. Thanks Damien!